config/config_test.go

```diff
@@ -531,6 +531,33 @@ func TestAvailableMemoryCmdLine(t *testing.T) {
 	assert.Equal(t, expected, inMemConfig.AvailableMemory)
 }
 
+func TestSendKeyCmdLine(t *testing.T) {
+	cm := makeYAML("General.ConfigurationVersion", 2, "AccessKeys.SendKey", "my-send-key")
+	rm := makeYAML("ConfigVersion", 2)
+	cfg, rules := createTempConfigs(t, cm, rm)
+	c, err := getConfig([]string{"--no-validate", "--config", cfg, "--rules_config", rules, "--send-key", "another-send-key"})
+	assert.NoError(t, err)
+
+	accessKeysConfig := c.GetAccessKeyConfig()
+	assert.Equal(t, "another-send-key", accessKeysConfig.SendKey)
+}
+
+func TestSendKeyEnvVar(t *testing.T) {
+	cm := makeYAML("General.ConfigurationVersion", 2, "AccessKeys.SendKey", "my-send-key")
+	rm := makeYAML("ConfigVersion", 2)
+	cfg, rules := createTempConfigs(t, cm, rm)
+
+	oldenv := os.Getenv("REFINERY_SEND_KEY")
```

Comment:
nit- we can use [t.Setenv](https://beta.pkg.go.dev/testing@master#T.Setenv) to take advantage of the automatic cleanup

Reply:
Fixed in https://github.com/honeycombio/refinery/pull/1323/commits/fcee2150196452418926781dac1456f6aeb25d0f 👍🏻